stdlib/3/unittest.pyi

```diff
@@ -15,8 +15,8 @@ _FT = TypeVar('_FT', Callable[[Any], Any])
 
 
 def skip(reason: str) -> Callable[[_FT], _FT]: ...
-def skipIf(condition: bool, reason: str) -> Callable[[_FT], _FT]: ...
-def skipUnless(condition: bool, reason: str) -> Callable[[_FT], _FT]: ...
+def skipIf(condition: object, reason: str) -> Callable[[_FT], _FT]: ...
+def skipUnless(condition: object, reason: str) -> Callable[[_FT], _FT]: ...
```

Comment:
I usually prefer leaving the type out, but I see your point about `__nonzero__` in this particular case.